osfclient/models/storage.py

```diff
@@ -1,10 +1,18 @@
 import os
+import six
 
 from .core import OSFCore
 from .file import ContainerMixin
 from .file import File
 from ..utils import norm_remote_path
 
+if six.PY2:
+    class FileExistsError(OSError):
+        """
+        Exception raised when a file already exists
+        Standard in Python 3
+        """
+
```

Comment:
Answered my own question by trying it out. Seems with the docstring you don't need a `pass`. Every day is a school day!